src/worker.js

```diff
@@ -54,6 +54,10 @@ const initServer = (app, config, callback) => {
       .catch(exit(code));
   }
 
+  function readyToClose(getClose) {
+    getClose(shutDownSequence);
```

Comment:
Thanks for the feedback!

Would be better to expose `close` or `shutDownSequence` in `worker` to `hypernova`?

One of the problems I had writing tests when exposing those was that sometimes I was trying to close the http server when it hasn't been initialised yet.